### PR TITLE
RT-549 Annotation checkbox disables the wrong annotation

### PR DESCRIPTION
**Pull request by amolht**
## Overview

- Fixed annotation checkbox disables the wrong annotation issue.

## How it was tested

- Tested it by launching charts-on-fhir locally.
- Ran unit test cases locally.

## Checklist

- [X] The title contains a short meaningful summary
- [X] I have added a link to this PR in the Jira issue
- [X] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [X] I have updated unit tests
- [X] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

**Review by yashpmohod — APPROVED** [review]

File: projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts [hunk 1]
```diff
@@ -22,17 +22,17 @@ export class DataLayerColorService {
...
   getAnnotationColor(annotation: any): string | undefined {
     const color = annotation.backgroundColor;
-    if (typeof color === 'string') {
-      return color;
+    if (typeof color === 'string' && !color.includes('33')) {
+      return color + '33';
     }
-    return undefined;
+    return color;
   }
```
Comment (semicolin) [inline projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts:36]: @amolht please revert this change to color service. 
1. The getter should not be changing the value that it gets.
2. `includes()` will match a '33' anywhere in the color string.
